package.json

```diff
@@ -23,5 +23,8 @@
   },
   "devDependencies": {
     "types-publisher": "Microsoft/types-publisher#production"
+  },
+  "dependencies": {
```

Comment:
This should be brought in as part of `types-publisher`. If you are having problems linting, please make a separate issue for that.